<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QWidget" name="gridWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="9" column="0">
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>300</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="error4">
           <property name="text">
            <string>Invalid file name!</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLabel" name="error6">
           <property name="text">
            <string>Invalid file name!</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Fourth Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of Warriors</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="text">
            <string>war5.txt</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="error3">
           <property name="text">
            <string>Invalid file name!</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="error8">
           <property name="text">
            <string>error</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="error1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Invalid file name!</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>First Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLineEdit" name="lineEdit_13">
           <property name="text">
            <string>war3.txt</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>8192</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Minimum Separation</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Second Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Fifth Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Third Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Core Size</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="error7">
           <property name="text">
            <string>error</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Default Settings</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="error5">
           <property name="text">
            <string>Invalid file name!</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Sixth Warrior File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Advanced Settings</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="lineEdit">
           <property name="text">
            <string>war1.txt</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="error2">
           <property name="text">
            <string>Invalid file name!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>372</width>
           <height>375</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="renderArea" name="rArea" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="speedDown">
      <property name="text">
       <string>Speed Down</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="speedUp">
      <property name="text">
       <string>Speed Up</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCOREWAR">
    <property name="title">
     <string>COREWAR</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuCOREWAR"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionDebug_mod">
   <property name="text">
    <string>Debug mod</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>renderArea</class>
   <extends>QWidget</extends>
   <header>/home/ulyssess/qtProjects/core_war/renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error7</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error8</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error1</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_13</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_14</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error4</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_15</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error5</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_11</sender>
   <signal>textChanged(QString)</signal>
   <receiver>error6</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
